site/content/en/docs/faq/_index.md

```diff
@@ -17,3 +17,7 @@ The easiest approach is to use the `docker` driver, as the backend service alway
 `none` users may want to try `CHANGE_MINIKUBE_NONE_USER=true`,  where kubectl and such will still work: [see environment variables]({{< ref "/docs/handbook/config.md#environment-variables" >}})
 
 Alternatively, configure `sudo` to never prompt for the commands issued by minikube.
+
+## How to ignore system verification?
+
```

Comment:
add this:
minikube's bootstrapper, [Kubeadm](https://github.com/kubernetes/kubeadm) verifies a list of features on the host system before installing kubernete. in case you get this error, and you still want to try minikube anyways despite your system limitation you can skip the verification by starting minikube with this extra option:
```shell
minikube --extra-config kubeadm.ignore-preflight-errors=SystemVerification
```

Reply:
`--ignore-preflight-errors` doesn't really skip the verification. It just displays the errors as warning instead.

> --ignore-preflight-errors stringSlice
  A list of checks whose errors will be shown as warnings. Example: ‘IsPrivilegedUser,Swap’. Value ‘all’ ignores errors from all checks.

Reply:
@medyagh updated. PTAL.

Reply:
@prasadkatti +1